app/logger/logger.js

```diff
@@ -1,29 +1,18 @@
-import { addColors, createLogger, format, transports } from 'winston'
+/*
+ *
+ * Levels:
+ * error: // Used for errors that prevent the application from operating correctly
+ * warn: // Used for errors that do not prevent the application from operating correctly, but may cause issues
+ * info: // Application access logs
+ * verbose: // Third party access logs
+ * debug: // Third party response logs
+ * silly: // Detailed function logs
+ */
+
+import { createLogger, format, transports } from 'winston'
 import { jsonStringify } from './utils.js'
 import { stackTraceFormatter } from './winstonFormatters.js'
 
-const levels = {
-  levels: {
-    health: 1,
-    error: 2,
-    warn: 3,
-    info: 4,
-    verbose: 5,
-    debug: 6,
-    silly: 7
-  },
-  colors: {
-    health: 'white',
-    error: 'red',
-    warn: 'yellow',
-    info: 'green',
-    verbose: 'cyan',
-    debug: 'blue',
-    silly: 'magenta'
```

Comment:
think trace would normally be log level beyond debug

Reply:
have removed this now, 'silly' is the standard log past debug for winston, can put this code back and modify if you prefer?

Reply:
just looked and looks like these are the default levels are:

```
error
warn
info
http
verbose
debug
silly
  ```

Ill have a rethink how i can use these better, thinking on the http level being useful

Reply:
have had another pass, hopfully we can set log level to `http`. Will still show request/response info without sensative data

Reply:
ahh cool if that is standard in winston then happy to go with it.